### ISSUE TITLE
NavX does not update on the Dashboard

### ISSUE THREAD

**Issue by Technologyman00** [BroncBotz3481/YAGSL#13]
The navX does not update on the Dashboard. It is just stuck at 0. 
![image](https://github.com/BroncBotz3481/YAGSL/assets/16891521/4a9ae7bf-afbe-47da-a410-2492b99ac19d)
Also Spams the VS Code console:
![image](https://github.com/BroncBotz3481/YAGSL/assets/16891521/62bf7f5f-b524-43ea-a625-f5f8ab68a044)

**Comment by Technologyman00** [comment]
I tried all the different dashboards, but no movement.

**Comment by Technologyman00** [comment]
I have not.

**Comment by Technologyman00** [comment]
> Any update? @Technologyman00

I haven't spent a lot of time on it, but I got the NavX to work in the Rev Swerve Example with no wiring or configuration change to the NavX. I changed a couple of lines of code on the Rev example as it doesn't by default support the NavX, but at a minimum there were no errors and it would display on the dashboard.

**Comment by Technologyman00** [comment]
It is not yet public, but its not working with YAGSL yet. I will probably work on it some more tonight.

**Comment by Technologyman00** [comment]
Still not working. The NavX is directly attached to the Rio through the MXP port. I preformed the reset instructions as described: https://pdocs.kauailabs.com/navx-mxp/guidance/gyroaccelcalibration/ and it did not change anything.  Here is our github: https://github.com/Trial-N-Terror-FRC-7900/YAGSL-Max-Swerve-Example/tree/main

**Comment by Technologyman00** [comment]
Any of the NavX example code works flawlessly

**Comment by Technologyman00** [comment]
So unfortunately due to Labor Day the school was closed and we were not able to test it out. However, previously I changed line 41 in swervelib/imu/NavXSwerve.java from `gyro = new AHRS(port);` to `gyro = new AHRS(SPI.Port.kMXP);` That didn't yield any difference. I will attempt the difference on Wednesday to see, but another theory is that we currently have the NavX MXP connected to both USB and the MXP port on the rio as in the event of a brownout the NavX will never loose power as the USB ports won't be disabled, so its possible it is confused about the different serial paths. So we will also try unplugging that as well.
